lib/src/maintenance.dart

```diff
@@ -367,6 +367,16 @@ Future<Maintenance> detectMaintenance(
         score: 10.0));
   }
 
+  // Checking the non-English characters in the description
+  if (nonAsciiRuneRatio(description) > 0.1) {
+    maintenanceSuggestions.add(new Suggestion.warning(
+        SuggestionCode.pubspecDescriptionAsciiOnly,
+        'The description contains too many non-ASCII characters.',
+        'The primary audience of this site will speak and use English, please '
+        'use fewer foreign characters in the description.',
+        score: 20.0));
```

Comment:
Code looks good to me.

@mit-mit Is this the semantics and language that we want?

Reply:
Let's change "The primary audience of this site will speak and use English, please use fewer foreign characters in the description" to "The site uses English as it's primary language. Please use a description that primarily contains characters used when writing English."